server/src/main/java/org/apache/iotdb/db/metadata/rescon/MemSchemaEngineStatistics.java

```diff
@@ -37,6 +38,8 @@ public class MemSchemaEngineStatistics implements ISchemaEngineStatistics {
 
   private final AtomicLong totalSeriesNumber = new AtomicLong(0);
 
+  private final AtomicInteger schemaRegionNumber = new AtomicInteger(0);
```

Comment:
There's a map storing all SchemaRegion in SchemaEngine. Can we use it to achieve such statistics.

Reply:
Use SchemaEngine to calculate schemaRegionNumber now.